tests/api.c

```diff
@@ -1617,7 +1617,63 @@ static void test_wolfSSL_CTX_SetTmpDH_buffer(void)
     wolfSSL_CTX_free(ctx);
 #endif
 }
+static void test_wolfSSL_DH_get0_pqg(void)
+{
+#if defined(OPENSSL_EXTRA) && !defined(NO_DH)
+    DH *dh = NULL;
+    BIGNUM* p;
+    BIGNUM* q;
+    BIGNUM* g;
+
+#if defined(OPENSSL_ALL)
+#if !defined(HAVE_FIPS) || (defined(HAVE_FIPS_VERSION) && (HAVE_FIPS_VERSION>2))
+    FILE* f = NULL;
+    unsigned char buf[4096];
```

Comment:
Does "buf" need to be 4096 here?  The dh2048.der file is only 268 bytes.

Reply:
Reduced to 268.